src/components/application_manager/src/application_data_impl.cc

```diff
@@ -555,6 +664,33 @@ bool DynamicApplicationDataImpl::IsSubMenuNameAlreadyExist(
   return false;
 }
 
+DataAccessor<WindowParamsMap>
+DynamicApplicationDataImpl::window_optional_params_map() const {
+  return DataAccessor<WindowParamsMap>(window_params_map_,
+                                       window_params_map_lock_ptr_);
+}
+
+void DynamicApplicationDataImpl::AddWindowInfo(
```

Comment:
I think "AddWindowInfo" implies that the info is appended to existing info. Because this method is overwriting window info for a specific window id, I think the method should be renamed "SetWindowInfo"

Reply:
fixed in 656568d5f8c8ff6c75ac80395d810fefe6c3a817